### PR TITLE
Try ipycanvas in Pyolite

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by jtpio** [comment]
Kicking RTD

**Comment by martinRenou** [comment]
It seems to work just great!

I only need to find a way to make `sleep` work in Python for this demo.

File: examples/ipycanvas.ipynb [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for _ in range(1_000):\n",
+    "    x = life_step(x)\n",
+    "    draw(x, canvas, '#5770B3')\n",
+    "\n",
+    "    sleep(0.1)"
```
Comment (jtpio) [inline examples/ipycanvas.ipynb:125]: Maybe we could event use `asyncio` here:

```python
import asyncio
await asyncio.sleep(0.1)
```

https://user-images.githubusercontent.com/591645/122605412-b3227e80-d077-11eb-8c1d-4ec9495dacfe.mp4

File: docs/environment.yml [hunk 2]
```diff
@@ -27,6 +27,7 @@ dependencies:
   # list of federated extensions for the demo site
   - ipywidgets
   - bqplot
+  - ipycanvas
```
Comment (jtpio) [inline docs/environment.yml:30]: Needs a quick rebase since #156 also modified this line.

Comment (martinRenou) [reply docs/environment.yml:31]: Fixed!

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by martinRenou** [comment]
Amazing! Thanks

**Pull request merged**